reactions/examples/validate.cpp

```diff
@@ -156,21 +148,32 @@ string line2;
 int Nk =  mypk.size();
 
 
-double p1,p2,p3;
+ double p1;
  //output file name
- const char* output = "mytest.dat";
+ const char* output = "p1h_real_z1.dat";
  /* Open output file */
  FILE* fp = fopen(output, "w");
 
- for(int i =0; i < Nk;  i ++) {
+ for(int i =0; i <Nk;  i ++) {
       real k = mypk[i][0];
       // quantity to validate
-      double react_q = halo.reaction_nu(k,vars);
-      p1 = react_q/mypk[i][1];
 
-//if(fabs(p1-1.)>0.001){
+         p1 = halo.reaction_nu(k,vars)/mypk[i][1];
+
+       // p1 = P_cb(k)/mypk[i][1];
+       // p1 = P_nu(k)/mypk[i][1];
+
+      // p1 = halo.one_halo(k,vars) / mypk[i][1];
+      // p1 = halo.one_halop(k,vars)/ mypk[i][1];
+
+      // p1 = halo.cvirialp(log10(k),vars[0])/ mypk[i][1];
+      // p1 = halo.nvirialp(log10(k),vars[1]-vars[6])/mypk[i][1];
+      // p1 = halo.rvirialp(log10(k),vars)/ mypk[i][1];
+
+//if(fabs(p1-1.)>0.002){
```

Comment:
are these needed?

Reply:
The criteria of 0.2% agreement or the various quantities? I guess we can keep theem there to make validation of alll quanttitties easier?